string object_name
string division
bool spawn_object